### ISSUE TITLE
send midi messages when a Bank is loaded

### ISSUE THREAD

**Issue by matthieusaque** [Morningstar-Engineering/MC6-MKII-Midi-Controller#18]
When I choose a BANK, it would be confortable to have the possibility to send Midi Messages when it's loaded (after a small lag, then it doesn't send several midi messages when banks are scrolled).

Then, when I choose a BANK, all midi devices are synced.

**Comment by jeffrutland** [comment]
yes!!  my main usage is Mainstage on a Mac, playing live.  I would love to be able to send a PC message when changing banks, so as to switch presets on Mainstage at the same time.  this functionality would make things SUPER awesome!!
